src/consensus/ledgerenclavetypes.h

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#pragma once
+
+#include "ds/ringbuffer_types.h"
+
+namespace consensus
+{
+  using Index = uint64_t;
+  /// Consensus-related ringbuffer messages
+  enum : ringbuffer::Message
+  {
+    /// Request individual log entries. Enclave -> Host
+    DEFINE_RINGBUFFER_MSG_TYPE(log_get),
```

Comment:
Since this is a refactoring/renaming PR and if it's not too tedious, could we rename these from `log_` to `ledger_`?

Reply:
ah good catch

Reply:
done